### PR TITLE
Add hexo-theme-aircv theme

### PR DESCRIPTION
**Pull request by molavec**
Change name from aircv-hexo-theme-aircv

## Check List

**Please read and check followings before submitting a PR.**

- [x] I want to publish my theme on Hexo official website.
    - [x] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.
    - [x] `preview` URL is correct.
    - [x] `preview` URL web site is rendered correctly.
    - [x] Add a screenshot to `source/themes/screenshots`.
    - [x] Screenshot filename is same as value of `name`.
    - [x] Screenshot size is `800 * 500`.
    - [x] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
    - Languages:
    - [ ] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [ ] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -1,8 +1,22 @@
+- name: Hexo theme aircv
```
Comment (SukkaW) [inline source/_data/themes.yml:1]:
```suggestion
- name: aircv
```

**Comment by molavec** [comment]
Hi Sukka. 

Ok I will change for this name. At first I didn’t want to use same name, because template is a third party designer. But I got his permission to adapt It to hexo.

Question: 

Could I change theme name and keep name in GitHub repository?

**Comment by SukkaW** [comment]
> Could I change theme name and keep name in GitHub repository?

The name of GitHub repo doesn't matter.

**Comment by molavec** [comment]
Name updated from "hexo-theme-aircv" to "Aricv"

**Pull request merged**